.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
 _site
+build/
```

Comment:
I'd suggest an out of source build instead.

Reply:
removed.